packages/integration/api-report.md

```diff
@@ -183,6 +184,19 @@ export function getBitbucketRequestOptions(
   headers: Record<string, string>;
 };
 
+// @public
+export function getGerritFileContentsApiUrl(
+  config: GerritIntegrationConfig,
+  url: string,
+): string;
+
+// @public
+export function getGerritRequestOptions(config: GerritIntegrationConfig):
+  | Record<string, string>
+  | {
+      headers: Record<string, string>;
+    };
```

Comment:
you could tighten this return type by declaring `{ headers?: Record<string, string> }` as its explicit return type

Reply:
Nice!